libs/filamat/src/shaders/CodeGenerator.cpp

```diff
@@ -181,6 +181,14 @@ utils::io::sstream& CodeGenerator::generateProlog(utils::io::sstream& out, Shade
     }
     generateDefine(out, "FILAMENT_EFFECTIVE_VERSION", effective_version);
 
+    if (material.stereoscopicType == StereoscopicType::INSTANCED) {
+        generateDefine(out, "FILAMENT_STEREO_INSTANCED", true);
+    } else if (material.stereoscopicType == StereoscopicType::MULTIVIEW) {
+        generateDefine(out, "FILAMENT_STEREO_MULTIVIEW", true);
+    } else {
+        assert(false);
+    }
```

Comment:
if you used a switch/case here you wouldn't need the assert

Reply:
Ah makes sense. Let me change it to switch.